code/modules/detectivework/scanner.dm

```diff
@@ -118,57 +207,67 @@
 							blood[blood_DNA] = blood_type
 
 		// We gathered everything. Create a fork and slowly display the results to the holder of the scanner.
-
-		var/found_something = 0
+		found_something = 0
 		add_log("<B>[station_time_timestamp()][get_timestamp()] - [target_name]</B>", 0)
+		if(advanced)
+			var/area/location_of_scan = get_area(A)
+			add_log("<B>Location of scan:</B> [location_of_scan.map_name].")
+			add_log("<B>GPS coordinate of scan:</B> [location_of_scan.x],[location_of_scan.y]")
```

Comment:
This is the location of the area, not the turf

Reply:
```suggestion
			add_log("<B>GPS coordinate of scan:</B> [get_turf(A).x],[get_turf(A).y]")
```